test/e2e/framework/service/jig.go

```diff
@@ -773,6 +773,11 @@ func testReachabilityOverNodePorts(nodes *v1.NodeList, sp v1.ServicePort, pod *v
 	internalAddrs := e2enode.CollectAddresses(nodes, v1.NodeInternalIP)
 	externalAddrs := e2enode.CollectAddresses(nodes, v1.NodeExternalIP)
 	for _, internalAddr := range internalAddrs {
+		// If the node's internal address points to localhost, then we are not
+		// able to test the service reachability via that address
+		if internalAddr == "127.0.0.1" {
+			continue
```

Comment:
Can we add a log line so it is clear this is skipped?

Reply:
Yes, added as suggested.